website/api/generating-proofs/deterministic-builds.md

```diff
@@ -0,0 +1,35 @@
+# Deterministic Guest Program Builds
+
+In the context of RISC Zero application development, [deterministic builds][term-deterministic-builds] are necessary to ensure a clear linkage between the source code for the [guest program][term-guest-program] and the resulting [Image ID][term-image-id].
+
+<!-- TODO: link to issue -->
```

Comment:
Do we have a good link to why we need docker laid out in detail? ideally in the form of an issue you can subscribe to, as I hope some day we might find a way to not require docker for determinism. :crossed_fingers:

Reply:
Yes, see: https://github.com/risc0/risc0/issues/116, which links to many rust/cargo issues related to this.